docs/fdc3-charter.md

```diff
@@ -7,7 +7,7 @@ title: FDC3 Charter
 Financial desktop applications include any app used in common financial workflows:
 
 * Traditional native applications implemented in C++, .NET, Java, Python, etc.
-* Hybrid web/native applications - stand alone native apps embedding Chromium (e.g. Electron, CEF, NW.js)
+* Hybrid web/native applications - stand alone native apps embedding Chromium (e.g. Electron, CEF, NW.js, FDC3-Sail)
 * Desktop web applications - platform based apps extending Chromium (e.g. OpenFin, Finsemble, Glue42)
```

Comment:
These two lines are already wrong and I think this edit is compounding it. We use the definition of hybrid web/native apps elsewhere and none of the listed examples actually fit it. Electron and FDC3-Sail should be on the next line (Desktop web applications), which itself needs a tweak, 

```suggestion
* Hybrid web/native applications - stand-alone native apps embedding Chromium (e.g. a .NET application embedding WebView using CEF or WebView2)
* Desktop web applications - Web applications running in a desktop container (e.g. OpenFin, Finsemble, Glue42, Electron, FDC3-Sail)
```